src/components/LoansByCategory/CategoryRow.vue

```diff
@@ -3,14 +3,24 @@
 		<div class="row title-row">
 			<div class="column small-12">
 				<h2 class="category-name">
-					<a class="view-all-link"
+					<router-link
```

Comment:
converted to router-link so we can hop to ui page without refresh!

Reply:
Nice